### DISCUSSION TITLE
Expand q-table row

### DISCUSSION THREAD

**Discussion by codesbyap** [quasarframework/quasar#11742] [General - Components / Directives / etc]
Hi,

I would like to implement a similar q-table with expand option on the right side of the screen. The expand button has to stretch itself and the q-tablerow vertically, to then show more details.

Would this be possible by using q-table?

Thanks in advance for answering/helping.

I also setup a codepen:
[Codepen setup](https://codepen.io/apdsgn/pen/GRMvZoz)

![image](https://user-images.githubusercontent.com/9142030/146851990-978de504-d1ab-485f-8b9f-48e8a5c1ffe4.png)

**Comment by MilosPaunovic** [comment]
You could add a class to expanded `row` to set its height.

[https://codepen.io/milospaunovic/pen/jOGLyOw](https://codepen.io/milospaunovic/pen/jOGLyOw)
